intro.tex

```diff
@@ -0,0 +1,15 @@
+
+\section{Introduction}
+\label{sec:intro}
+
+The Verification Control Document (VCD) is a detailed look at where we are in verifying our requirements e.g. \citeds{LSE-61}.
+We verify requirements by
+running tests, the results of which are recorded in Jira.  A description of how the verification testing is set up is given in Section 2 of \citeds{LDM-503}.
+Specifically Section 2.3.2 of \citeds{LDM-503} describes the verification elements, test cases which appear in \secref{sec:vcd}.
+
+In general the summary provided in \secref{sec:summary} will give the reader the overview required, the remainder of the document is a large matrix of all the DM requirements , their verification elements, tests and if those tests have run.
```

Comment:
Replace "the overview required, the remainder" with "an overview.  The remainder".  Replace "tests and if those tests have run" with "their tests, and if those tests have been run".